app/views/submissions/_form.html.erb

```diff
@@ -63,8 +67,26 @@
 					<%= t('submission.no_manuscript_yet') %>
 				</span>
 			</div>
-	    <div class="col-12" id="userManuscript-informer">
-  	  </div>
+	    <div class="col-12" id="userManuscript-status">
+			<div class="col-12" id="file-size-correct" style="display:none"><i class="fas fa-check"></i> <%=t('submission.correct_file_size')%></div>
+			<div class="col-12" id="file-size-too-big" style="display:none"> <i class="fas fa-exclamation-triangle"></i> <%=t('submission.file_size_too_big')%></div>
+			<div class="col-12" id="file-size-way-too-big" style="display:none"><i class="fas fa-bomb"></i> <%=t('submission.file_size_way_too_big')%></div>
+			<div class="col-12" id="file-optimize" style="display:none">
+			<small><%=raw t("submission.optimization_help")%><%= helpdesk(t('submission.optimization_info'),true)%></small>
+			<div class="col-12">
+			<button id="userManuscript-optimize-btn" class="btn btn-sm btn-outline-secondary"><%= t('submission.optimize') %></button>
```

Comment:
I would prefer `btn-primary` instead of `btn-outline-secondary` here. The reason is that the file upload button `#userManuscript-uploadButton-btn` is already `btn-outline-secondary`. To give the user more guidance what the next step in the workflow is, an optically different button might be more expedient. As the 'Save'-Button is in the same color but deactivated at that point in time, this would make sense to me.

Also, `submission.optimize` is not yet defined in the locale files.